### DISCUSSION TITLE
Are there documented comparisons between Eradiate and other software?

### DISCUSSION THREAD

**Discussion by Srheft** [eradiate/eradiate#337] [General]
Hi Eradiate users/developers,

I was wondering if there are "worked through" examples of comparison between Eradiate and other commonly used software (e.g. MODTRAN, S6, etc) available. I imagine before Eradiate's public release many sanity checks were done to ascertain correct behavior of the kernel at various initial conditions but they may not be extensively documented and/or publicly available; which in that case, a note on what would be the simplest yet correct checkpoint that would make a fair comparison would be helpful. Changing the default molecular makeup [such as water vapor] to extreme levels and compare the 'brf' for flat spectral surfaces comes to mind but one worry is that naive approaches in creating "test atmospheres" results in unfair comparison/conclusions.

Appreciate experienced members' guidance on this,
S.

**Comment by leroyvn** [comment]
Hi @Srheft, as of now, there is no public documented comparison between Eradiate and other radiative transfer models.

Our 3D surface simulation features are validated against [ROMC test cases](https://romc.jrc.ec.europa.eu/_www/). We also participate in the [RAMI-V](https://rami-benchmark.jrc.ec.europa.eu/_www/phase_descr.php?strPhase=RAMI5) benchmark.

For the atmospheric simulation components, we participate in the [RAMI4ATM benchmark](https://rami-benchmark.jrc.ec.europa.eu/_www/phase_descr.php?strPhase=RAMI4ATM). Internally, at Rayference, we use test cases derived from RAMI4ATM specifications to compare Eradiate with a trusted reference model called RTMOM (which we also used to participate in RAMI4ATM), which uses a different RTE integration method (matrix operator), implements a similar molecular absorption modelling technique (correlated k-distribution) and uses an independent source of input data. However, for a detailed comparison, we will all have to wait for the publication of RAMI4ATM results and this will take time.

**Reply by Srheft** [reply]
Thank you for this informative response.
